e2e/iam/atlas_users_test.go

```diff
@@ -49,8 +51,56 @@ func TestUsers(t *testing.T) {
 		if err := json.Unmarshal(resp, &users); err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
+
 		if len(users) == 0 {
 			t.Fatalf("expected len(users) > 0, got %v", len(users))
 		}
+
+		username = users[0].Username
+		userID = users[0].ID
+	})
+
+	t.Run("Describe by username", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			usersEntity,
+			"describe",
+			"--username",
+			username,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		}
+
+		var user mongodbatlas.AtlasUser
+
+		if err := json.Unmarshal(resp, &user); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+	})
+
+	t.Run("Describe by id", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			iamEntity,
+			usersEntity,
+			"describe",
+			"--id",
+			userID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		}
+
+		var user mongodbatlas.AtlasUser
+
+		if err := json.Unmarshal(resp, &user); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
```

Comment:
[nit] similar to above, right now we only check it doesn't error but we should also check the data, at least the id matches